src/components/Navigation/components/Section/tests/Section.test.tsx

```diff
@@ -253,6 +253,35 @@ describe('<Navigation.Section />', () => {
     });
   });
 
+  it('displays the itemsPrefix when passed', () => {
+    const section = mountWithNavigationProvider(
+      <Section
+        title="test"
+        items={[
+          {
+            label: 'some label',
+            url: '/admin',
+            disabled: false,
+          },
+        ]}
+        action={{
+          icon: 'placeholder',
+          accessibilityLabel: 'This is a test section',
+          onClick: noop,
+        }}
+        itemsPrefix={
+          <div testID="ItemsPrefix">
```

Comment:
Try and avoid testID where possible, searching based upon a class name for this sort of thing should be fine

Reply:
👍